### ISSUE TITLE
TOPIC: Menu items must be active all the way across

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#439]
I just clicked on "Edit topic" but landed in the space just to the right of the text. This closed the menu and didn't open what I wanted. So for better UX they should be clickable all the way across, not just where the text is (and for all menus, not just this one) :) 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/964c550f-ff75-4339-80a8-715b4a3fd409)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish fixed the issue

**Comment by ssin1901** [comment]
@ilmartyrk @BeccaMelhuish  tested and yes, I clicked on the space next to Edit which I have indicated in screenshot1 with a red dot. This closed the entire menu which was a bit annoying.

Do both of you feel that the menu should stay open unless the user clicks out of the menu?

<img width="1364" alt="Screenshot 2024-02-04 at 15 48 56" src="https://github.com/citizenos/citizenos-fe/assets/72747107/6083700f-9c81-4a6d-a889-3a6390f28ac7">
